ansible/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: ENV
+
+virtualenv_path = ./virtualenv
+
+ENV: Makefile
+	virtualenv2 --distribute $(virtualenv_path)
+
+CHECK_ENV:
+ifndef VIRTUAL_ENV
+	$(error PLEASE ENTER THE VIRTUALENV BEFORE FUN THE COMMAND)
```

Comment:
`FUN`?

Reply:
That was `RUNNING`, no idea why it become `FUN`, I'll fix it